client/src/layout/Footer/Footer.js

```diff
@@ -1,7 +1,149 @@
 import React from 'react';
+import SmartphoneOutlinedIcon from '@mui/icons-material/SmartphoneOutlined';
+import LocationOnOutlinedIcon from '@mui/icons-material/LocationOnOutlined';
+import TimerOutlinedIcon from '@mui/icons-material/TimerOutlined';
+import FacebookRoundedIcon from '@mui/icons-material/FacebookRounded';
+import PinterestIcon from '@mui/icons-material/Pinterest';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import YouTubeIcon from '@mui/icons-material/YouTube';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import { Box, Typography, Link as MUILink } from '@mui/material';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import scrollToTop from '../../utils/scrollToTop/scrollToTop';
+import styles from './Footer.module.scss';
+
+const navigationMenu = [
+  {
+    name: 'Home',
+    path: '/',
+  },
+  {
+    name: 'Catalogue',
+    path: '/catalogue',
+  },
+  {
+    name: 'Cart',
+    path: '/cart',
+  },
+];
+
+const footerGallery = [
+  {
+    imagePath: './assets/images/footer/CARPATHIANS.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/CHERNIVTSI.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/IVANOFRANKIVSK.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/KYIV.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/KYIVCENTER.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/PRYKARPATTIA.png',
+    altDescription: 'recent_trips_image',
+  },
+];
 
 const Footer = () => {
-  return <div></div>;
```

Comment:
if I'm not mistaken, we can not add an empty <div></div> but take semantic quotes or return something in some other way

Reply:
it`s not empty, please, review the last commit